### PR TITLE
Add number of current connections per user

### PR DESCRIPTION
**Pull request by CharlesJUDITH**
This change will add:
- The number of current connections per user
- Apply linter
- Use python3

### CODE REVIEW

File: mysql.py [hunk 1]
```diff
@@ -471,17 +481,44 @@ def fetch_mysql_process_states(conn):
...
+    row = result.fetchone()
+    return bool(row['Value'] == 'ON')
+
+
+def fetch_connections_per_account(conn):
+    """ Fetch number of connections per account """
+    queries = {}
+    try:
+        result = mysql_query(conn, """
+		        SELECT user, current_connections
```
Comment (khodin) [inline mysql.py]: this query returns one line per user, host couple and user can be null.
i think we should aggregate the results to have all the data :

```
SELECT user, sum(current_connections) as `current_connections`
FROM performance_schema.accounts
WHERE user is not null
GROUP BY user
```

File: mysql.py [hunk 2]
```diff
@@ -641,6 +685,11 @@ def read_callback():
     for key in slave_status:
         dispatch_value('slave', key, slave_status[key], 'gauge')
 
+    if is_ps_enabled(conn) is True:
+        user_connections = fetch_connections_per_account(conn)
+        for key in user_connections:
+            dispatch_value('connection_per_user', key, user_connections[key], 'counter')
```
Comment (khodin) [inline mysql.py]: as this query get current connections, wouldn't this value be of gauge type ?

Comment (CharlesJUDITH) [reply mysql.py]: Indeed gauge is more appropriate for this metric.

File: mysql.py (hunk 1 above)
Comment (CharlesJUDITH) [reply mysql.py]: Indeed

**Review by khodin — APPROVED** [review]

**Pull request merged**